contracts/ethregistrar/BaseRegistrar.sol

```diff
@@ -1,51 +1,32 @@
 pragma solidity ^0.8.4;
 
 import "../registry/ENS.sol";
+import "./IBaseRegistrar.sol";
 import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-abstract contract BaseRegistrar is Ownable, IERC721 {
-    uint constant public GRACE_PERIOD = 90 days;
+abstract contract BaseRegistrar is Ownable, IERC721, IBaseRegistrar {
+    uint256 public constant GRACE_PERIOD = 90 days;
 
     event ControllerAdded(address indexed controller);
     event ControllerRemoved(address indexed controller);
-    event NameMigrated(uint256 indexed id, address indexed owner, uint expires);
-    event NameRegistered(uint256 indexed id, address indexed owner, uint expires);
-    event NameRenewed(uint256 indexed id, uint expires);
+    event NameMigrated(
+        uint256 indexed id,
+        address indexed owner,
+        uint256 expires
+    );
+    event NameRegistered(
+        uint256 indexed id,
+        address indexed owner,
+        uint256 expires
+    );
+    event NameRenewed(uint256 indexed id, uint256 expires);
 
     // The ENS registry
     ENS public ens;
```

Comment:
Maybe this can be immutable?

Reply:
We can't redeploy this, so we shouldn't make meaningful changes to it.

Reply:
Do you mean we can't redeploy `BaseRegistrar.sol`? If so, why do we make any changes to the file at all?